### PR TITLE
TC-872 spog-UI config

### PR DESCRIPTION
**Pull request by desmax74**
See https://issues.redhat.com/browse/TC-872

### CODE REVIEW

**Review by ctron — CHANGES_REQUESTED** [review]
IIRC there are some more locations of this

File: spog/api/src/config/default.yaml [hunk 1]
```diff
@@ -417,6 +421,10 @@ cve:
                     end.toLocaleString("default", { month: "2-digit" }) + "-" +
                     end.toLocaleString("default", { day: "2-digit" })
                   ];
+              - id: "2023"
+                label: "2023"
+                terms:
+                  - 'created:2023-01-01..2024-01-01'
```
Comment (ctron) [inline spog/api/src/config/default.yaml:427]: The qualifier is named `datePublished` in this case.

File: deploy/k8s/trustification-branding/files/spog-ui.yaml [hunk 2]
```diff
@@ -411,6 +419,10 @@ cve:
                     end.toLocaleString("default", { month: "2-digit" }) + "-" +
                     end.toLocaleString("default", { day: "2-digit" })
                   ];
+              - id: "2023"
+                label: "2023"
+                terms:
+                  - 'created:2023-01-01..2024-01-01'
```
Comment (ctron) [inline deploy/k8s/trustification-branding/files/spog-ui.yaml:425]: The qualifier is named `datePublished` in this case.

File: deploy/k8s/trustification-branding/files/spog-ui.yaml [hunk 3]
```diff
@@ -287,6 +291,10 @@ vexination:
...
```
Comment (ctron) [inline deploy/k8s/trustification-branding/files/spog-ui.yaml:297]: The qualifier is named `release` in this case.

File: spog/api/src/config/default.yaml [hunk 4]
```diff
@@ -296,7 +296,7 @@ vexination:
               - id: "2023"
                 label: "2023"
                 terms:
-                  - 'created:2023-01-01..2024-01-01'
```
Comment (desmax74) [inline spog/api/src/config/default.yaml]: @ctron last missing fix

**Pull request merged**